File: FileOperation.cs
using System;
using System.Linq;

namespace FileOperation
{
    class Program
    {
        static void Main(string[] args)
        {
            bool isBear = false;
            const string FILE_PATH = "textfileBears.txt";

            //retrieves the first line from a text file.
            string lineOne = System.IO.File.ReadLines(FILE_PATH).First();
            //splits the first line into individual words.
            string[] splitLine = lineOne.Split();
            //searches through the array of words looking for the word "bear."
            foreach(string word in splitLine)
            {
                //when the word "bear." is found isBear becomes true .
                if(word == "bear.")
                {
                    isBear = true;
                    break;
                }
                
            }
            //when isBear is true it prints that the word bear is present if its false it prints saying that it's not.
            if (isBear)
            {
                Console.WriteLine("The line contains the word \"bear.\"");
            }
            else
            {
                Console.WriteLine("This line doesn't contain the word \"bear.\"");
            }
            
        }
    }
}

File: Javascript/FileOperation.js
const readline = require('readline');
const fs = require('fs');
const file = "bear.txt"

//creates a way to read data from a file.
const readInterface = readline.createInterface({
    input: fs.createReadStream(file),
    console: false
});

//reads data from a file line by line
readInterface.on('line',function(line){
    let isBear = false;
    //split the line into individual words
    let str = line.split(' ');
    //loop through the words looking for bear.
    for(let i = 0; i < str.length; ++i){
        //if the word bear. is found isBear becomes true.
        if(str[i] === 'bear.'){
            isBear = true;
            break;
        }
    }
    if(isBear)
        console.log("bear. is present in that line");
    else 
        console.log("bear. is not present in that line");
    //closes the filestream.
    readInterface.close();
});